src/main/java/jalau/at18/katas/bankocr/daniela/Separatedigits.java

```diff
@@ -0,0 +1,29 @@
+package jalau.at18.katas.bankocr.daniela;
+
+public class Separatedigits extends AnalyzedDigit {
+    private static final int ZISE = 3;
```

Comment:
Be careful with typos (*SIZE)

Reply:
Already correct de SIZE word to the SIZEONEDIGIT